mycroft/skills/core.py

```diff
@@ -234,6 +234,11 @@ def __init__(self, skill):
         self.page = None    # the active GUI page (e.g. QML template) to show
         self.skill = skill
         self.on_gui_changed_callback = None
+        self.config_core = Configuration.get()
```

Comment:
let's just call this member `config`. I know it doesn't match the one in the MycroftSkill class but that will be changed from self.config_core to self.config as part of 19.08